charts/manager/values.yaml

```diff
@@ -1,16 +1,21 @@
+affinity: {}
+nodeSelector: {}
+tolerations: []
 replicaCount: 1
 revisionHistoryLimit: 3
+
 containerRegistry: CONTAINER_REGISTRY_TO_REPLACE
 image: IMAGE_TO_REPLACE
 hash: HASH_VERSION_TO_REPLACE
-resources:
-  limits:
-    memory: '50Mi'
-    cpu: '50m'
+
+resources: {}
+  # limits:
+  #   memory: '50Mi'
+  #   cpu: '50m'
```

Comment:
any reason to keep the commented code?

Reply:
yes.
this file has the default values for the Helm chart. It's better to avoid fixing resources limits/allocation by means of default values, to force the user to choose their own (it's a widespread practice)

Reply:
I meant: shouldn't we remove the code intead of keeping it commented?